cmd/monaco/account/deploy.go

```diff
@@ -142,7 +142,7 @@ func deploy(fs afero.Fs, opts deployOpts) error {
 	return nil
 }
 
-func createAccountClients(manifestAccounts map[string]manifest.Account) (map[deployer.AccountInfo]*accounts.Client, error) {
+func CreateAccountClients(manifestAccounts map[string]manifest.Account) (map[deployer.AccountInfo]*accounts.Client, error) {
```

Comment:
this func might need another home, for now i've just left it where it is (cmd package)

Reply:
Weren't there some similar utils in a `cmd/monaco/dynatrace` package at some point? Might fit there